webapp/src/containers/addOrEditSubscriptions/subComponents/eventsPanel.tsx

```diff
@@ -53,52 +55,71 @@ const EventsPanel = forwardRef<HTMLDivElement, EventsPanelProps>(({
         >
             <div className='padding-h-12 padding-v-20 wizard__body-container'>
                 <div className='events-panel__prev-data border-radius-8 padding-v-10 padding-h-25 margin-bottom-25'>
-                    <h4 className='events-panel__prev-data-header font-14 wt-400 margin-0'>{'Channel'}</h4>
-                    <p className='events-panel__prev-data-text font-14 wt-400 margin-v-5'>
-                        {channel?.label}
-                    </p>
-                    <h4 className='events-panel__prev-data-header font-14 wt-400 margin-top-15 record-header'>{`Record${subscriptionType === SubscriptionType.BULK ? ' type' : ''}`}</h4>
-                    <p className='events-panel__prev-data-text font-14 wt-400 margin-v-5'>{subscriptionType === SubscriptionType.RECORD ? record : RecordTypeLabelMap[recordType]}</p>
+                    <div className='d-flex'>
+                        <div>
+                            <h4 className='events-panel__prev-data-header font-14 wt-400 margin-top-15 record-header'>{'Channel'}</h4>
+                            <p className='events-panel__prev-data-text font-14 wt-400 margin-v-5'>
+                                {channel?.label}
+                            </p>
+                            <h4 className='events-panel__prev-data-header font-14 wt-400 margin-top-15 record-header'>{`Record${subscriptionType === SubscriptionType.BULK ? ' type' : ''}`}</h4>
+                            <p className='events-panel__prev-data-text font-14 wt-400 margin-v-5'>{subscriptionType === SubscriptionType.RECORD ? record : RecordTypeLabelMap[recordType]}</p>
+                        </div>
+                        <div className='margin-left-150'>
+                            {filters.map((filter) => (
+                                filter.filterValue &&
+                                    <div key={filter.filterValue ?? ''}>
```

Comment:
```suggestion
                                    <div key={filter.filterValue}>
```
Not needed because of the condition on above line.